examples/suit_update/README.hardware.md

```diff
@@ -306,10 +305,10 @@ see 6 pairs of messages indicating where (filepath) the file was published and
 the corresponding coap resource URI
 
     ...
-    published "/home/francisco/workspace/RIOT/examples/suit_update/bin/samr21-xpro/suit_update-riot.suitv3_signed.1557135946.bin"
-           as "coap://[2001:db8::1]/fw/samr21-xpro/suit_update-riot.suitv3_signed.1557135946.bin"
-    published "/home/francisco/workspace/RIOT/examples/suit_update/bin/samr21-xpro/suit_update-riot.suitv3_signed.latest.bin"
-           as "coap://[2001:db8::1]/fw/samr21-xpro/suit_update-riot.suitv3_signed.latest.bin"
+    published "/home/francisco/workspace/RIOT2/examples/suit_update/bin/samr21-xpro/suit_files/riot.suit.1632124156.bin"
```

Comment:
Nitpick, put to avoid confusion 

```suggestion
    published "/home/francisco/workspace/RIOT/examples/suit_update/bin/samr21-xpro/suit_files/riot.suit.1632124156.bin"
```

Reply:
will do

Reply:
done

Reply:
done